src/Command/FeedImportCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Command;
+
+use App\Message\FeedItemDataMessage;
+use App\Repository\FeedRepository;
+use App\Services\Feeds\Mapper\FeedConfigurationMapperService;
+use App\Services\Feeds\Parser\FeedParserInterface;
+use CuyZ\Valinor\Mapper\MappingError;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+#[AsCommand(
+    name: 'app:feed:import',
+    description: 'Parse feed and import events from it',
+)]
+class FeedImportCommand extends Command
+{
+    public function __construct(
+        private readonly MessageBusInterface $messageBus,
+        private readonly FeedParserInterface $feedParser,
+        private readonly FeedConfigurationMapperService $configurationMapperService,
+        private readonly FeedRepository $feedRepository,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument('feedId', InputArgument::REQUIRED, 'Database feed id')
+            ->addOption('limit', '', InputOption::VALUE_REQUIRED, 'Limit the number of items outputted', -1);
+    }
+
+    /**
+     * @throws MappingError
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
Think it's simple load feed config loop item and send theme into message system. I can move it to at service if you insist, but it will be a 5-8 lines function yielding progress back.